en/deploy-tidb-cluster-across-multiple-kubernetes.md

```diff
@@ -57,7 +57,7 @@ Run the following command:
 {{< copyable "shell-regular" >}}
 
 ```bash
-cat << EOF | kubectl apply -f -n ${cluster1_namespace} -
+cat << EOF | kubectl apply -n ${cluster1_namespace} -f -
```

Comment:
@july2993 Please also help update the file in `zh` dir, thanks!

Reply:
https://github.com/pingcap/docs-tidb-operator/pull/1167/commits/4f4e1e32c73f64d7664ad5c13c3cdc7bc6c8da0c